### ISSUE TITLE
[firebase_ui_localizations] Turkish Translation Mistake

### ISSUE THREAD

**Issue by atillaturkmen** [firebase/flutterfire#11338]
## Bug report

The Turkish translation for the "registerActionText" and "registerText" texts are incorrect in the `firebase_ui_tr.arb` file. The current translation says "Kaydet" for "registerActionText" and "Kaydedin" for "registerText" but the correct translations should be "Kaydol" and "Kaydolun" respectively.

### Steps to reproduce

1. Open the application in Turkish language.
2. Navigate to the registration page or perform any registration action.
3. Observe the text for the "registerActionText" and "registerText" strings.

### Expected behavior

The expected behavior is that the "registerActionText" should be translated as "Kaydol" and the "registerText" message should be translated as "Kaydolun."

**Comment by danagbemava-nc** [comment]
I see the lines here https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/src/lang/tr.dart#L171 & here https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/src/lang/tr.dart#L177

**Comment by atillaturkmen** [comment]
> I see the lines here
> 
> https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/src/lang/tr.dart#L171
> 
> & here
> https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/src/lang/tr.dart#L177

Yes these are the lines i changed in the [relevant PR](https://github.com/firebase/flutterfire/pull/11339).

**Comment by atillaturkmen** [comment]
> I see the lines here
> 
> https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/src/lang/tr.dart#L171
> 
> & here
> https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/src/lang/tr.dart#L177

I have changed these lines as well as this line 
https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/l10n/firebase_ui_tr.arb#L263
and this line
https://github.com/firebase/flutterfire/blob/d98b24fe21f5a02552aedac4ef4db45cda02520e/packages/firebase_ui_localizations/lib/l10n/firebase_ui_tr.arb#L273

in the [relevant PR](https://github.com/firebase/flutterfire/pull/11339).
